src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestSCMHead.java

```diff
@@ -36,12 +36,31 @@
 
     private static final long serialVersionUID = 1;
 
-    public PullRequestSCMHead(int number) {
+    private String user;
```

Comment:
Well, `user` is actually the top-level field; `login` is just a subfield of that, but ambiguous with `merged_by.login`, `assignee.login`, `repo.owner.login`, ad nauseam. So I think `user` is correct here.